enterprise/server/invocation_search_service/invocation_search_service.go

```diff
@@ -48,23 +57,101 @@ func defaultSortParams() *inpb.InvocationSort {
 	}
 }
 
-func (s *InvocationSearchService) rawQueryInvocations(ctx context.Context, sql string, values ...interface{}) ([]*tables.Invocation, error) {
-	rows, err := s.h.RawWithOptions(ctx, db.Opts().WithQueryName("search_invocations"), sql, values...).Rows()
+// Clickhouse UUIDs are formatted without any dashes :(
```

Comment:
It's too bad we didn't keep the original format... Can you add some bounds checks for len(uuid)?

Reply:
this always winds up padded to length == 32, so just added a check for that.